lib/srv/db/cloud/watchers/rds.go

```diff
@@ -173,7 +182,12 @@ func (f *rdsAuroraClustersFetcher) Get(ctx context.Context) (types.Databases, er
 
 // getAuroraDatabases returns a list of database resources representing RDS clusters.
 func (f *rdsAuroraClustersFetcher) getAuroraDatabases(ctx context.Context) (types.Databases, error) {
-	clusters, err := getAllDBClusters(ctx, f.cfg.RDS, common.MaxPages)
+	clusters, err := getAllDBClusters(ctx, f.cfg.RDS, common.MaxPages, false)
+	if common.IsUnrecognizedAWSEngineNameError(err) {
+		f.log.WithError(err).Warn("Teleport supports an engine which is unrecognized in this AWS region. Querying engine versions.")
+		// fallback to try to describe only clusters with engines recognized in the AWS region.
+		clusters, err = getAllDBClusters(ctx, f.cfg.RDS, common.MaxPages, true)
```

Comment:
What would you say about moving the fallback logic inside `getAllDBClusters` ?

Reply:
That's where I put the fallback logic in the first place, but then I wanted to log the error if it happens... so if I move it in there I need to pass the fetcher's logger in as well